analyze/index.ts

```diff
@@ -162,7 +188,7 @@ export async function detectSensitiveInfo(
   detect?: DetectSensitiveInfoFunction,
 ): Promise<SensitiveInfoResult> {
   const { log } = context;
-  const coreImports = createCoreImports(detect);
+  const coreImports = createCoreImports(detect || noOpDetectSensitiveInfo);
```

Comment:
```suggestion
  const coreImports = createCoreImports(detect);
```
Doesn't need this if defaulted, right?